src/message.rs

```diff
@@ -638,7 +638,6 @@ mod tests {
 
         {
             let connect = message.command.connect.as_ref().unwrap();
-            assert_eq!(connect.client_version, "2.0.1-incubating");
```

Comment:
Let's update the assertion instead of remove it?

Reply:
1. follow the same rule of generation
2. doesn't malformed (i.e., ` env!("CARGO_PKG_VERSION")` returns not empty or even in pattern `X.Y.Z`)

Reply:
The assertion is based on the bytes array input of a message which we won't update for each version of pulsar-rs? WDYT?

Not sure to understand your second comment?

Reply:
LGTM. Ty!